client/src/components/new-request/send-to-examination.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <MainContainer id="non-analyze-container">
+    <template v-slot:container-header>
+      <v-col cols="auto" class="h6 pt-0 mt-1">
+        You are searching for a name for a
+        {{ entityText === ' BC Corporation' && location.text === ' BC' ? '' : ' ' + location.text }}
+        {{ entityText }}
+      </v-col>
+    </template>
+    <template v-slot:content>
+      <v-row class="mt-n2">
+        <NameInput :is-read-only="true"/>
+      </v-row>
+      <v-row class="text-center mt-n8">
```

Comment:
That's an awful lot of negative margin... It's likely better to reduce margins on the other containers involved than to go this negative here.

My rule of thumb is "up to n2 or so" for alignment, otherwise it's a container problem that needs to be solved.

Reply:
Great point, i'll look into this.

Reply:
Fixed